src/app/routes/enrolment/enrolment-list/enrolment-list.component.ts

```diff
@@ -268,4 +274,27 @@ export class EnrolmentListComponent implements OnInit, OnDestroy {
 
     this.loadingService.hide();
   }
+
+  private _checkNamespaceControlChanges(): void {
+    if (!this.namespaceFilterControl) {
+      return;
+    }
+
+    this.namespaceFilterControl.valueChanges
+        .pipe(
+            distinctUntilChanged((prevValue, currentValue) => prevValue === currentValue),
+            takeUntil(this._subscription$)
+        )
+        .subscribe((filterValue) => {
```

Comment:
you don't need here `{ ...}`. It's fine to write `(filterValue) => this_updateList(filterValue)`